framework/src/main/java/org/checkerframework/common/basetype/BaseTypeValidator.java

```diff
@@ -183,10 +183,9 @@ protected void reportValidityResultOnUnannotatedType(
             final @CompilerMessageKey String errorType,
             final AnnotatedTypeMirror type,
             final Tree p) {
-        // TODO: if underlying is a compound type such as List<@Palindrome String>, then it would be
-        // nice to print it as "List" instead of as "List<@Palindrome String>".
-        TypeMirror underlying = type.getUnderlyingType();
-        checker.report(Result.failure(errorType, type.getAnnotations(), underlying.toString()), p);
+        TypeMirror unannotated =
```

Comment:
The variable name `unannotated` is now a bit unfitting, as it is also the erased type.
How about keeping `underlying` as before? Or `reportType`?

Reply:
I will keep it as `underlying`.